src/import-data/useImportRequest.ts

```diff
@@ -146,6 +171,11 @@ export const getImportRequest = (queryParams: QueryParams): Promise<ImportReques
       return Promise.resolve(getFileImportRequest(queryParams, 'entities'));
     case 'tdrexport':
       return getTDRSnapshotExportImportRequest(queryParams);
+    case 'tdrexportmanifest':
```

Comment:
Another approach here would be to combine this with `tdrexport` and allow the `tdrmanifest` query param to be missing, and if so, generate the manifest on demand. That might be cleaner since they do map to the same downstream import operation, once the manifest is generated.

Reply:
Yeah, actually that feels better to me.